OWNERS_ALIASES

```diff
@@ -152,6 +152,7 @@ aliases:
   - evankanderson
   - gerardo-lc
   - kvmware
+  - mgencur
```

Comment:
Just in case you missed some contexts - currently adding a person to the reviewers group does not have an effect, due to the issue described in https://github.com/knative/test-infra/issues/2942